portable/Zynq.2019.3/ff_sddisk.c

```diff
@@ -504,7 +504,10 @@ FF_Disk_t * FF_SDDiskInit( const char * pcName )
         /* Initialise the created disk structure. */
         memset( pxDisk, '\0', sizeof( *pxDisk ) );
 
-        pxDisk->ulNumberOfSectors = myCSD.sd_last_block_address + 1;
+        /* Apparently is `myCSD.sd_last_block_address` is not correct/accurate */
```

Comment:
Hi @htibosch ,

I have managed to incorporate this PR into my own build and this change is breaking my system.  It seems pxSDCardInstance->SectorCount is being set to zero... This in turn causes the mount to fail.

I am unable to dig into why this is the case until the middle of next week as I am going to be away until Wednesday.  I am willing to investigate it further on my return and report back here when I find out what is happening if that is helpful.

Let me know your thoughts when you can.

Kind Regards,

Pete

Reply:
Hi @htibosch ,

It seems my development environment is somewhat broken right now please dis-regard my previous comments as it is clearly a problem at my end, I am really sorry for all the noise!  

When I return from by trip next week I will be reinstalling all my systems, hopefully this will fix the issues.

Kind Regards,

Pete